src/components/common/UploadView/index.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import { useRef } from 'react';
+
+import Plus from 'public/assets/icon/plus.svg';
+
+export default function UploadView() {
+  const imageUploadButtonRef = useRef<HTMLInputElement | null>(null);
```

Comment:
혹시 null도 제네릭에 포함시킨 이유가 있을까요?👀
아래처럼 사용하면 RefObject로 추론됩니당!
```suggestion
  const imageUploadButtonRef = useRef<HTMLInputElement>(null);
```

Reply:
아하! 무턱대고 썼네요 ㅎㅎ.. 저도 좀 공부해서 상황에 맞게 타입 변경하도록 하겠습니다!